changelog/_unreleased/2020-09-17-add-ability-to-search-by-custom-field-in-administration-backend.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Add ability to search by Custom Fields in Administration Backend
+issue: NEXT-10855
+author: Mroczny Czesiek, Wojciech Milek
```

Comment:
Unfortunately we only support one author at the moment, also for `author_email` and `author_github`

Reply:
This still needs to be done for the `author`, which this comment was originally pointing at. Sorry for any confusion!